pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/store/PirDatabase.kt

```diff
@@ -48,17 +51,47 @@ import com.squareup.moshi.Types
         ScanNavigateResult::class,
         ScanErrorResult::class,
         ExtractProfileResult::class,
+        UserProfile::class,
     ],
 )
 @TypeConverters(PirDatabaseConverters::class)
 abstract class PirDatabase : RoomDatabase() {
     abstract fun brokerJsonDao(): BrokerJsonDao
     abstract fun brokerDao(): BrokerDao
     abstract fun scanResultsDao(): ScanResultsDao
+    abstract fun userProfileDao(): UserProfileDao
 
     companion object {
+        private val MIGRATION_1_2 = object : Migration(1, 2) {
```

Comment:
Given this is not in production, do we need the actual migration. Can we just do destructive migration? No strong opinions though, maybe there's a good reason